react/tic-tac-toe/src/services/AuthService.js

```diff
@@ -1,6 +1,18 @@
+import { wrapService, withPostSuccess } from 'redux-recompose';
+
 // eslint-disable-next-line import/no-unresolved
 import api from '~config/api';
 
-export default {
-  login: credentials => api.post('/login', credentials)
+const login = credentials => api.post('/login', credentials);
+
+login.successSelector = response => response.data.token;
+
+login.injections = [
+  withPostSuccess((_, response) => localStorage.setItem('token', response.data.token))
+];
+
+export const service = {
+  login
 };
+
+export default wrapService(service, 'auth', { login: 'auth' });
```

Comment:
create this `auth` in a constants file in de redux folder, something like `REDUCER_TARGET`
and the other one `auth` as a `AUTH_TARGET` and use it anywhere you use this reducer.